common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -286,9 +286,6 @@ def test_problem_type_a11y(self):
 
         self.problem_page.a11y_audit.config.set_rules({
             "ignore": [
-                'aria-allowed-attr',  # TODO: AC-491
-                'aria-valid-attr',  # TODO: AC-491
-                'aria-roles',  # TODO: AC-491
                 'checkboxgroup',  # TODO: AC-491
```

Comment:
Removing these last 3 ignores introduce test failures. Checkboxgroup and radiogroup ignores are related to the funkier radio/checkbox + text input field problems. Removing section introduces a number of different types of failures-- FYI @clrux, as AC-491 has been closed.